PWGGA/GammaConv/AliAnalysisTaskGammaCaloMergedML.cxx

```diff
@@ -273,7 +272,11 @@ AliAnalysisTaskGammaCaloMergedML::AliAnalysisTaskGammaCaloMergedML(): AliAnalysi
   fMergedClusterTreeXPos(0),
   fMergedClusterTreeYPos(0),
   fMergedClusterTreeIsPrimary(0),
-  fMergedClusterTreePID(0)
+  fMergedClusterTreePID(0),
+  fMergedClusterTreeEta(0),
```

Comment:
for the future consider saving Px,Py,Pz and E which determines whole 4-vector and saves storage. Pt, eta, phi can then be determined "offline"